### ISSUE TITLE
Is not present anymore in Firefox webstore 

### ISSUE THREAD

**Issue by robdearpowa** [DominicMaas/TimelineExtension#94]
Is not possible to find the extension  in Firefox add-ons store

**Comment by abulawright** [comment]
Seems to be back up! https://addons.mozilla.org/en-GB/firefox/addon/timeline-support-for-firefox/?src=search

**Comment by lwittwer** [comment]
That does seem to be the same extension but it is not the official release.

**Comment by d1vanloon** [comment]
@lwittwer You are correct. I've publishing under that listing in an attempt to keep the tool available for Firefox users.

**Comment by martinimarcello00** [comment]
Is it possible to install the extension manually?

**Comment by NatoBoram** [comment]
If it's not officially on the Firefox add-ons website anymore, and someone else had the time to re-publish it, has @DominicMaas abandoned the project?

**Comment by DominicMaas** [comment]
I have not much use for the project anymore (I don't use Chrome or Firefox anymore, using the new Edge instead). And Microsoft released an official extension. 

I have to decouple logins from my server eventually, so if anyone wants to take over, let me know.

I will try clean up and republish the extension soon, I just kept getting annoyed with Firefox pulling down the extension all the time.

**Comment by NatoBoram** [comment]
> And Microsoft released an official extension.

Wait, did they? Where is it?

**Comment by d1vanloon** [comment]
The referenced official extension is available from the [Chrome Web Store](https://chrome.google.com/webstore/detail/web-activities/eiipeonhflhoiacfbniealbdjoeoglid#cw). There is no official extension for Firefox.
